.github/workflows/tests_gpu_cuda.yml

```
@@ -119,7 +119,7 @@ jobs:
 
       - name: Install required packages
         run: |
-          python -m pip install ninja cmake custatevec-cu${{ matrix.cuda_version }}
+          python -m pip install ninja "cmake!=3.29.0" custatevec-cu${{ matrix.cuda_version }}
```

Could you also add some comment about this change for future reference?

Maybe something like,
 ```suggestion
          # Omitting the installation of cmake v3.29.0 due to
          # https://github.com/scikit-build/cmake-python-distributions/pull/474
          python -m pip install ninja "cmake!=3.29.0" custatevec-cu${{ matrix.cuda_version }}
```